tests/ui/proc-macro/helper-attr-compat-collision.rs

```diff
@@ -0,0 +1,15 @@
+//@ proc-macro: test-macros.rs
+//@ proc-macro: extra-empty-derive.rs
+//@ build-pass
```

Comment:
```suggestion
//@ check-pass
```
We are not testing any linking or codegen here.